cloudformation/template.yml

```diff
@@ -0,0 +1,339 @@
+Description: A CloudFormation template for deploying Raster Vision Batch jobs to AWS.
+
+Metadata:
+  'AWS::CloudFormation::Interface':
```

Comment:
Interesting. I was not aware that this existed. This should help out the unfamiliar user quite a bit.